Dockerfile

```diff
@@ -81,7 +81,11 @@ RUN echo "source activate gatk" > /root/run_unit_tests.sh && \
 
 RUN cp -r /root/run_unit_tests.sh /gatk && \
     cp -r /root/gatk.jar /gatk
-ENV CLASSPATH=/gatk/gatk.jar:$CLASSPATH PATH=$CONDA_PATH/envs/gatk/bin:$CONDA_PATH/bin:$PATH
+ENV CLASSPATH=/gatk/gatk.jar:$CLASSPATH IN_GATKLITE_DOCKER=true PATH=/gatk:$PATH
```

Comment:
Is it possible to split these across multiple lines for ease of reading?  Not a whole new env block for each, I'm not sure if Dockerfiles let you use \\ or something for line continuations.